style.css

```diff
@@ -0,0 +1,299 @@
+    padding: 15px 50px;
+    font-size: 30px;
+    color: #fff;
+    margin-bottom: 40px;
+}
+
+.btn {
+    margin: 10px 14px;
+    padding: 15px 50px;
+      font-size: 30px;
+}
+
+ a {
+    text-decoration: none;
+}
+.nav-bar {
+    color: dodgerblue;
+}
+
+
+.aside-img {
+  width: 200px;
+  height: 200px;
+  border-radius: 50%;
+  
+}
+
+
+
+
+.home main{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    
+    position: relative;
+    min-height: 100vh;
+    background-color: var(--dark-beige);
+}
+.home main::before, .home  main::after{
+    content: "";
+height: 100%;
+position: absolute;
+right: 0;
+display: block;
+width: 55%;
+background-color: #FFF;
+margin-left: auto;
+
+
+z-index: 1;
+}
+.text-box h2 {
+    font-size: 27px;
+    font-weight: 800;
+    margin: 25px auto 15px auto;
+    padding-bottom: 12px;
+    width: fit-content;
+    border-bottom: 1px solid var(--royal-blue);
+}
+.text-box h3{
+    font-size: 17px;
+    margin-bottom: 51px;
+    font-weight: 400;
+}
+.text-box{
+    padding-top: 30px;
+    text-align: center;
+}
+
+
+.main-box {
+    position:  relative;
+    z-index: 2;
+    width: 100%;
+    max-width: 800px;
+    margin: 0 auto;
+     display: grid;
+    grid-template-columns: 50% 50%;
+    align-items: center;
+}
+
+.main-box > section:nth-child(1){
+    background-color: var(--dark-beige);
+    box-shadow: -11px 11px 17px 0px rgba(138,131,124,0.23);
+    
+}
+
+
+.l-b {
+    background-color: #FFF;
+}
+
+
+/* second css  */
+.flex {
+    display: flex;
+}
+.text-sm {
+    font-size: 17px;
+    font-weight: 300;
+}
+.aside h2 {
+    font-weight: 800;
+    font-size: 100px;
+text-align: left;
+}
+.aside h4 {
+   font-size: 25px; 
+   margin-bottom: 20px;
+    text-align: left;
+    font-weight: 500;
+}
+.aside p {
+    margin-bottom: 20px;
+    font-weight: 300;
+    font-size: 17px;
+    text-align: left;
+}
+.btn-container {
+gap: 20px;
+padding-bottom:  40px;
+
+}
+.btn-1, .rc-header a {
+    background-color: #0150FD;
+    color: white;
+border-radius: 25px;
+padding: 8px 30px;
+text-transform: uppercase;
+font-weight: 500;
+text-decoration: none;
+}
+.btn-2 {
+    border: 2px solid black;
+    color: black;
+border-radius: 25px;
+padding: 8px 30px;
+text-transform: uppercase;
+font-weight: 500;
+text-decoration: none;
+transition: 0.3s;
+}
+.btn-2:hover {
+    background-color: #0150FD;
+    color: #FFF;
+}
+
+/* resume  */
+.r {
+    padding: 70px 20px;
+    background-color: var(--light-beige);
+}
+.justify-b {
+    justify-content: space-between;
+}
+.align-c {
+align-items: center;
+}
+.r-header, .r-content, .rc-header {
+    max-width: 700px;
+    margin: 30px auto;
+}
+.rc-header h2 {
+    font-size: 25px;
+    font-weight: 700;
+}
+.grid-rc {
+    display: grid;
+    grid-template-columns: 25% 60%;
+    justify-content: space-between;
+    background-color: #fff;
+    margin-bottom: 30px;
+    padding: 67px 40px;
+}
+.grid-rc  h6 {
+    padding: 7px;
+}
```

Comment:
This is naturely quite responsive to the point that most desktops and laptops should be OK with it but the goal was to build something mobile-first, starting with the mobile layout and using media queries to adopt to larger screen sizes